packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -54,7 +54,16 @@ export const floatingLocalVideoModalStyle = (
     {
       main: localVideoTileContainerStyle(theme, isNarrow)
     },
-    { main: { boxShadow: theme.effects.elevation8 } }
+    {
+      main: {
+        boxShadow: theme.effects.elevation8,
+        ':focus-within': {
+          boxShadow: theme.effects.elevation16,
+          border: `2px solid ${theme.palette.neutralPrimary}`
```

Comment:
do we want to use rem here instead?

Reply:
updated this here to use rem, will add a task to update this on main as well.